e2e/features/provide-appointee-details.feature

```diff
@@ -0,0 +1,26 @@
+@migrated-to-exui @nightly-test
+Feature: The lapse
+
+  Background:
+    Given I presetup an "PIP" SYA case
+    And I am signed in as a Case Officer
+    And I navigate to an existing case
+
+  @provide-appointee-details
+  Scenario: Should end up in "With DWP" state when ALL fields are present
+    When I choose "Lapse appeal"
```

Comment:
@bensonlievoco Not sure if we need the lapse appeal steps as they mention a different functionality.
All we need to do is create a new case and add appointee details to the case and verify if they are added against the case

Reply:
Just updated